src/azure-cli/azure/cli/command_modules/role/tests/latest/test_graph.py

```diff
@@ -559,25 +559,59 @@ class GraphUserScenarioTest(ScenarioTest):
     def test_graph_user_scenario(self):
         self.kwargs = {
             'user1': self.create_random_name(prefix='graphusertest', length=20),
+            'user2': self.create_random_name(prefix='graphusertest', length=20),
             'domain': 'AzureSDKTeam.onmicrosoft.com',
-            'new_mail_nick_name': 'graphusertest',
+            'mail_nickname': 'graphusertest',
+            'new_mail_nick_name': 'graphusertestupdate',
             'group': 'graphusertest_g',
-            'pass': 'Test1234!!'
+            'password': 'Test1234!!',
+            'force_change_password_next_login': True,
         }
         # create
         user1_result = self.cmd(
-            'ad user create --display-name {user1} --password {pass} --user-principal-name {user1}@{domain}',
-            checks=[self.check("displayName","{user1}")]
+            'ad user create --display-name {user1} '
+            '--mail-nickname {mail_nickname} '
+            '--password {password} '
+            '--force-change-password-next-login {force_change_password_next_login} '
+            '--user-principal-name {user1}@{domain} ',
+            checks=[
+                self.check("displayName","{user1}"),
+                self.check("userPrincipalName", "{user1}@{domain}")
+            ]
```

Comment:
Please make sure all properties like `-mail-nickname` are also checked.

Reply:
--mail-nickname not returned by the result or the show command.